compiler/testData/codegen/boxInline/anonymousObject/twoCapturedReceivers/kt8668_nested.kt

```diff
@@ -0,0 +1,14 @@
+// FILE: 1.kt
```

Comment:
@pyos Could you add test that would make sense for AbstractJvmIrAgainstOldBoxInlineTest/AbstractJvmOldAgainstIrBoxInlineTest execution?

(Not sure but looks like  `f` should be also `inline` function and called from another file with some additional lambda)

Reply:
If it would be additional this$0 clash on 'f' inlining it would be the best test

Reply:
I don't think it's possible to get a descriptor collision unless the declaration and call are in the same class.

Reply:
If descriptor collision is not possible I think it worth to test inlining in mixed modes to be sure that additional renaming in ir scheme doesn't break old one on attempt to inline it

Reply:
Done